### PR TITLE
Add gems dotenv and annotate

### PR DESCRIPTION
**Pull request by mmarusyk**
**CHECK LIST**
- [ ] specs added
- [ ] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [ ] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [ ] screenshot of feature is added if task has UI

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -0,0 +1,3 @@
+OSBB_DATABASE_USERNAME = place_your_username_here
```
Comment (sviatoslav-krupa) [inline .env.template:1]: don't use spaces at '='

**Review by sviatoslav-krupa — APPROVED** [review]

**Comment by chu4** [comment]
make annotation in the end of a file pls

**Review by Ukhanskyi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ bundle && yarn
 rails db:create db:migrate db:seed
 ```
 
+### Rename file `.env.template` to `.env` and add values in environment variables
+
```
Comment (vasya539) [inline README.md:40]: не ренейм, copy file .env.template and rename it to .env, якщо ти заренеймиш оригінал то гіт буд думати що його видалили, так хтось і запушить, в результаті мають бути оба цих фала

File: config/database.yml [hunk 3]
```diff
@@ -17,6 +17,9 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: <%= ENV['OSBB_DATABASE_USERNAME'] %>
+  password: <%= ENV['OSBB_DATABASE_PASSWORD'] %>
+  host: <%= ENV['OSBB_DATABASE_HOST'] %>
```
Comment (vasya539) [inline config/database.yml:22]: хост не потрібно, він по дефолту у всіх localhost

**Review by DenisRem — APPROVED** [review]

**Review by marikrebega — APPROVED** [review]

**Review by chu4 — APPROVED** [review]

**Pull request merged**